src/Mailer/Transport/TwilioTransport.php

```diff
@@ -0,0 +1,49 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Copyright 2010 - 2022, Cake Development Corporation (https://www.cakedc.com)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright Copyright 2010 - 2022, Cake Development Corporation (https://www.cakedc.com)
+ * @license MIT License (http://www.opensource.org/licenses/mit-license.php)
+ */
+namespace CakeDC\Users\Mailer\Transport;
+
+use Cake\Mailer\AbstractTransport;
+use Cake\Mailer\Message;
+use Twilio\Rest\Client;
+
+class TwilioTransport extends AbstractTransport
```

Comment:
Do we need this tranport class? I see twilio is required in composer but I don't see when we are using this in the code.

Reply:
We do, this is the default transport class provided to users who wants otp with SMS out of the box. They can always implement a new transport or use email of course.